### ISSUE TITLE
rtn city still showing with L/O pax

### ISSUE THREAD

**Issue by ali206** [206Tours/DB#321]
should return city still show as a mandatory field if land only?

![image](https://cloud.githubusercontent.com/assets/8440903/7005837/027a40ae-dc4a-11e4-9458-0b86d3a1accf.png)

Sammie finished this order from incomplete orders and this is how it showed up originally (i updated invoice and now it looks ok but no idea why it would like like below):

![stef](https://cloud.githubusercontent.com/assets/8440903/7005860/3232ffb6-dc4a-11e4-9016-191c34406bda.png)

**Comment by bradatkins** [comment]
Not really, but it's always been like this. I can change it but it doesn't hurt anything I don't think

**Comment by ali206** [comment]
If it doesn't hurt anything then if someone is LO i would have that field remove itself b/c there's no need for it to be there and it would probably confuse our clients registering themselves. 

If that didn't cause the issue with the booking above then what did?
